shell/browser/electron_download_manager_delegate.cc

```diff
@@ -30,6 +30,15 @@
 #include "shell/common/gin_converters/file_path_converter.h"
 #include "shell/common/options_switches.h"
 
+#if BUILDFLAG(IS_WIN)
+#include "base/i18n/case_conversion.h"
+#include "base/win/registry.h"
+#include "ui/base/l10n/l10n_util.h"
+
+#define IDS_APP_SAVEAS_ALL_FILES 35936
+#define IDS_APP_SAVEAS_EXTENSION_FORMAT 35937
```

Comment:
These both come from `ui/strings/grit/ui_strings.h` - we can just include that since we shouldn't hardcode grit strings :)